src/main/java/seedu/address/model/food/Food.java

```diff
@@ -84,23 +84,23 @@ public boolean equals(Object other) {
 
         Food otherFood = (Food) other;
         return otherFood.getName().equals(getName())
-                && otherFood.getPhone().equals(getPhone())
+                && otherFood.getPrice().equals(getPrice())
                 && otherFood.getEmail().equals(getEmail())
                 && otherFood.getTags().equals(getTags());
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, tags);
+        return Objects.hash(name, price, email, tags);
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
-                .append(" Phone: ")
-                .append(getPhone())
+                .append(" Price: ")
```

Comment:
Maybe can add dollar sign when displaying price:
.append(" Price: $")

Reply:
Have made the changes in [this commit](https://github.com/AY1920S1-CS2103T-F13-2/main/pull/75/commits/c56568a6be094653d109905bb600107cf31a1ac5), thanks!